x-pack/plugins/security_solution/server/lib/detection_engine/fleet_integrations/api/get_installed_integrations/installed_integration_set.ts

```diff
@@ -49,7 +49,7 @@ export const createInstalledIntegrationSet = (): IInstalledIntegrationSet => {
     }
 
     // Actual `installed_version` is buried in SO, root `version` is latest package version available
-    const installedPackageVersion = fleetPackage.savedObject.attributes.install_version;
+    const installedPackageVersion = fleetPackage?.savedObject?.attributes.install_version || '';
```

Comment:
@criamico Thank you for addressing my previous comment. The only question left is related to `fleetPackage`. It doesn't look like `fleetPackage?.` required as `fleetPackage` isn't optional so you can remove the first optional chaining.

Reply:
Thanks Maxim, I'll address this tomorrow.